docs/content_management/data_migration/importing_data.md

```diff
@@ -22,33 +22,54 @@ Migrations store execution metadata in the `ibexa_migrations` database table.
 This allows incremental upgrades:
 the `ibexa:migrations:migrate` command ignores files that it had previously executed.
 
+## Migration step
+
+A data migration step is a single operation in data migration process
+that combines a mode (for example: `create`, `update`, `delete`)
+and a type (for example: `content`, `section`, `currency`),
+with optional additional information depending on the specific step.
+
+In a migration file, a step is an array item starting with the mandatory properties `type` and `mode`, for example:
+
+```yaml
+-
+    type: content
+    mode: create
+```
+
+Then, the step is described by additional properties depending on its type and mode.
+
+* See [Available migrations](#available-migrations) for the modes available for each type.
+* See [Migration examples](#migration-examples) to explore what you can do with each type.
+* For a custom migration step, see [Create data migration step](create_data_migration_step.md).
+
 ## Available migrations
 
-The following data migration steps are available:
-
-|                      | `create` | `update` | `delete` |
-|----------------------|:--------:|:--------:|:--------:|
-| `content`            | &#10004; | &#10004; | &#10004; |
-| `content_type`       | &#10004; | &#10004; | &#10004; |
-| `role`               | &#10004; | &#10004; | &#10004; |
-| `content_type_group` | &#10004; | &#10004; | &#10004; |
-| `user`               | &#10004; | &#10004; |          |
-| `user_group`         | &#10004; | &#10004; | &#10004; |
-| `language`           | &#10004; |          |          |
-| `object_state_group` | &#10004; |          |          |
-| `object_state`       | &#10004; |          |          |
-| `section`            | &#10004; | &#10004; |          |
-| `location`           |          | &#10004; |          |
-| `attribute_group`    | &#10004; | &#10004; | &#10004; |
-| `attribute`          | &#10004; | &#10004; | &#10004; |
-| `customer_group`     | &#10004; | &#10004; | &#10004; |
-| `currency`           | &#10004; | &#10004; | &#10004; |
-| `product_price`      | &#10004; |          |          |
+The following data migration step modes are available:
+
+| `type`                 | `create` | `update` | `delete` |
```

Comment:
Can we use alphabetical order in this table?

Reply:
@MagdalenaZuba OK.

Reply:
Done in 9a71b6912a5f3cc483c4576bf3b7eb9815728754